src/screens/InputUsername/index.js

```diff
@@ -275,7 +300,14 @@ const ChooseUsername = () => {
             <Text style={styles.title}>{StringConstant.onboardingChooseUsernameHeadline}</Text>
             <Text style={styles.desc}>{StringConstant.onboardingChooseUsernameSubHeadline}</Text>
             <View style={styles.containerInput}>
-              <TouchableOpacity style={styles.containerAddIcon} onPress={() => onPhoto()}>
+              <TouchableOpacity
+                style={styles.containerAddIcon}
+                onPress={() => {
+                  AnalyticsEventTracking.eventTrack(
+                    BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_CLICKED
+                  );
+                  onPhoto();
+                }}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider creating a separate function for event tracking to reduce redundancy.

```diff
+  const handleProfilePicClick = () => {
+    trackEvent(BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_CLICKED);
+    onPhoto();
+  }

  <TouchableOpacity
    style={styles.containerAddIcon}
    onPress={() => {
-     AnalyticsEventTracking.eventTrack(
-       BetterSocialEventTracking.ONBOARDING_USERNAME_PROFILE_PIC_CLICKED
-     );
-     onPhoto();
+     handleProfilePicClick();
    }}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->